authors/apps/articles/extra_views.py

```diff
@@ -69,4 +78,77 @@ def list(self, request):
                     'message': 'No articles bookmarked yet.'
                 },
                 status=status.HTTP_200_OK
-            )
\ No newline at end of file
+            )
+
+
+class ArticleAPIView(APIView):
```

Comment:
The original views file was over the recommended 250 lines long. So to avoid making it too long while adding our changes we moved it to the new extra views file. Codeclimate recomends doing this.

Reply:
If this is the problem you aim to solve, the better way would be to create a views folder and have every model's view in its own file in the folder.

Reply:
Ok, Should I go ahead and do that within this PR?